python-wrapper/src/neo4j_viz/nvl.py

```diff
@@ -66,7 +72,23 @@ def render(
         );
         """
         full_code = self.library_code + js_code
+
+        # Although a `<style>` element within the `HTML` block would work,
+        # it is not standard to use it outside of a `<head>` element.
+        # For this reason, the style for the buttons is inlined here.
+        # The combination of `onmouseover` and `onmouseout` replaces the
+        # `button:hover` descriptor.
+        button_style = """
+            style="background-color: #0a6190; color: #fff; margin: 0 0 1rem 0; padding: .85em 1em; border: 1px solid transparent; border-radius: 5px; text-align: center; font-family: 'Public Sans', 'Helvetica Neue', helvetica, roboto, arial, sans-serif; font-weight: 500; font-size: 14px"
+            onmouseover="this.style.backgroundColor='#014063'"
+            onmouseout="this.style.backgroundColor='#0a6190'"
+        """
+
         html_output = f"""
+        <button type="button" {button_style} onclick="{nvl_varname}.nvl.saveToFile({{ filename: '{download_name}' }})">Save screenshot</button>
```

Comment:
the saved png will have the ideal ratio for notebook output cells.
But i assume cropping the image can be done as an aftermath.

Ideally we could save svg 🤔

Reply:
I can try using the auto-fit zoom level as I was doing manually in a prototype, and see if it's better.

For the SVG I guess we have to ask NVL devs.